docs/web/docs/1-getting_started/7-github.mdx

```diff
@@ -171,22 +177,31 @@ By installing IAMbic GitHub App, the app can integrate the pull-request and merg
 
 ### Store the following secrets in AWS Secrets Manager in the same AWS account running GitHub App lambda function.
 
+1. Go to AWS Secret Manager, the URL will look like `console.aws.amazon.com/secretsmanager/listsecrets`. 
+	1. Consider check the region, the default used in the Terraform is `us-west-2`.
 1. Store the earlier downloaded private key contents (not the filename) in AWS Secrets Manager and name it `iambic/github-app-private-key`.
-(This is the default name used in the Terraform that will be executed later. If you use another name, be sure to update it in the Terraform procedure later.)
+	1. This is the default name used in the Terraform that will be executed later. If you use another name, be sure to update it in the Terraform procedure later.
 
 <!-- # TODO: Mention plaintext etc. -->
 
-2. Generate a webhook secret. (e.g., by taking the output of `ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'` at the terminal).
-Store the webhook secret in AWS Secrets Manager and name it `iambic/github-app-webhook-secret`.
+2. Generate a webhook secret by taking the output of one of these commands: 
+	1. `ruby -rsecurerandom -e 'puts SecureRandom.hex(20)'`at the terminal.
+	1. `python -c "import secrets; print(secrets.token_hex(20))"`
+	1. `node -e "var token = require('crypto').randomBytes(20).toString('hex'); console.log(token)"`
+3. Store the webhook secret in AWS Secrets Manager and name it `iambic/github-app-webhook-secret`.
 
 ### Create an IAM role for Lambda function execution
 <!-- # TODO: Where do I put this file and how do I write it? -->
 
+1. Place the above IAMbic template to your `iambic-templates` repo under `resources/github/iam_role_lambda.yaml`
 
 ```yaml
 template_type: NOQ::AWS::IAM::Role
 included_accounts:
-  - aws_hub_account_name
+	# You will need to replace the "NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole" 
+	# with the account name of the AWS account holding the IambicHubRole. 
+	# In the event you change the secret names, you need to update those values as well.
+  - aws_hub_account_name 
```

Comment:
can we update this from `aws_hub_account_name` to `NAME_OF_AWS_ACCOUNT_THAT_HAS_IambicHubRole`